lib/jormungandr/test/data/jormungandr/stake_pools/create-stake-pool.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script (re)generates the required keys and certificates to setup a
+# stake-pool.
+#
+# Generated keys and certificates are checked in to git, and can manually be
+# inlined in genesis.yaml.
+#
+# The files are stored in a folder such that multiple stake-pools can be setup
+# in a convenient way.
+#
+# Usage:
+# Run in the stake_pools directory, where owner.prv exists.
+#
+# To create a stake pool in a folder "a":
+# .create-stake-pool.sh a
+
+echo "creating certificates for a new stake pool in the folder $1"
+mkdir $1
+cd $1
+
+jcli key generate --type=Curve25519_2HashDH > stake_pool_vrf.prv
+cat stake_pool_vrf.prv | jcli key to-public > stake_pool_vrf.pub
+jcli key generate --type=SumEd25519_12 > stake_pool_kes.prv
+cat stake_pool_kes.prv | jcli key to-public > stake_pool_kes.pub
+
+jcli certificate new stake-pool-registration \
+    --kes-key $(cat stake_pool_kes.pub) \
+    --vrf-key $(cat stake_pool_vrf.pub) \
+    --serial 1010101010 > stake_pool.cert \
```

Comment:
This `jcli certificate` command has two `>` output redirections. It may or may not work as you expect. Needs to be fixed.

Reply:
Done.